frontends/web/src/routes/account/walletconnect/dashboard.tsx

```diff
@@ -37,14 +39,25 @@ type TProps = {
 export const DashboardWalletConnect = ({ code, accounts }: TProps) => {
   const { t } = useTranslation();
   const { web3wallet, isWalletInitialized } = useContext(WCWeb3WalletContext);
+  const config = useLoad(getConfig);
   const [sessions, setSessions] = useState<SessionTypes.Struct[]>();
   const receiveAddresses = useLoad(getReceiveAddressList(code));
+  const navigate = useNavigate();
+
 
   const updateSessions = useCallback(() => {
     const activeSessions = Object.values(web3wallet?.getActiveSessions() || []);
     setSessions(activeSessions);
   }, [web3wallet]);
 
+  useEffect(() => {
+    if (config && config.frontend && !config.frontend.hasUsedWalletConnect) {
+      setConfig({ frontend: { hasUsedWalletConnect: true } }).then(() => {
+        navigate(0); // reloads the page
```

Comment:
This works in webdev, but the Qt app errors. Reason (I think) is that on-load Qt finds `index.html` and the `location.href` is something like `qrc:/index.html`, but on-reload the location is something like `qrc:/account/v0-12345678-eth-0/wallet-connect/dashboard'`.

![Screen Shot 2023-10-02 at 12 56 31](https://github.com/digitalbitbox/bitbox-wallet-app/assets/546900/3571f42e-981f-44bd-af62-8e32367fd638)

just thinking out loud: could the part that loads the library in the provider be moved to a function "load" and also exposed by the provider so that "load" can be called from here?

I guess that could work, but I have no idea if and how the web3wallet would be updated.

If that works somehow, I am thinking if the setConfig part should be moved to "load" function instead of here.